webapp/src/store/Store.spec.ts

```diff
@@ -1,7 +1,24 @@
+import { ProjectStore } from '@/store/ProjectStore';
 import { Store } from './Store';
 import { describe, expect, it } from '@jest/globals';
 
 describe('Store', () => {
+  it('get ProjectStore for a path', async () => {
+    const store = new Store();
+    const projectStore = new ProjectStore({
+      getTranslations: async () => ({
+        de: {
+          'greeting.headline': {
+            sourceFile: '',
+            text: 'Hallo',
+          },
+        },
+      }),
+    });
+    store.addProjectStore('repoPath', 'projectPath', projectStore);
+    const actual = store.getProjectStore('repoPath', 'projectPath');
+    expect(actual).toEqual(projectStore);
```

Comment:
Ok, but then I want a test that verifies that `getProjectStore` after `updateTranslation` returns an updated `ProjectStore`. With just a 'get ProjectStore for a path' that uses `toEqual`, the implementation could return a clone of ProjectStore. No test would fail but `translations/.../[msgId]` would not work.

Reply:
Agree, for now I will change it basicly to toBe() then later when we have a redis or similar solution we will rewrite the unit test

Reply:
Resolved in 0511e6019a9787bd5d1476499e65187288869a3b by using `toBe` instead of `toEqual`.